pymisp/api.py

```diff
@@ -1127,37 +1127,37 @@ def get_sharing_groups(self):
 
     def _set_user_parameters(self, **kwargs):
         user = {}
-        if kwargs.get('email'):
+        if kwargs.get('email'): 
             user['email'] = kwargs.get('email')
-        if kwargs.get('org_id'):
+        if kwargs.get('org_id') > 0:
             user['org_id'] = kwargs.get('org_id')
-        if kwargs.get('role_id'):
+        if kwargs.get('role_id') > 0:
             user['role_id'] = kwargs.get('role_id')
         if kwargs.get('password'):
             user['password'] = kwargs.get('password')
-        if kwargs.get('external_auth_required'):
+        if kwargs.get('external_auth_required') is not None:
```

Comment:
Yes, the main (only?) reason to check `None` is when the value can be `False`. 
Example:

``` python
if kwargs.get('termsaccepted') is not None: 
    user['termsaccepted'] = kwargs.get('termsaccepted')
```

With `kwargs['termsaccepted'] == False` and we want to pass that `False` to the sever.